designdocs/CAPACITY.md

```diff
@@ -0,0 +1,404 @@
+# Capacity Design Doc
+
+## Context and Scope
+
+Feeless transactions are important in reaching mass adoption as it removes the overhead costs of transactions for app developers to acquire a far reaching user base.
+
+In this document I will introduce the concept of Capacity and how capacity can be used to pay for transaction fees.
+
+**What is Capacity?**
+
+Capacity is a non-transferable resource that is associated to an MSA account. You can use capacity to pay for transactions and it replenishes after a certain period which can be used to continue to pay for new transactions. More details on how capacity replenishes below.
+
+## Proposal
+
+MRC has two types of account systems—a MSA account and a token account. You can acquire Capacity by staking your own tokens to your MSA account or when others stake their tokens to your MSA. The amount staked is reserved from an account free balance which is refunded after a thaw period. Once staked, Capacity is granted at a 1 to 1 conversion.  Upon receiving Capacity, it can then be used to pay for transactions. 
```

Comment:
```suggestion
Frequency has two types of account systems—a MSA account and a token account. You can acquire Capacity by staking your own tokens to your MSA account or when others stake their tokens to your MSA. The amount staked is locked from an account free balance which is unlocked after a thaw period. Once staked, Capacity is granted at a 1 to 1 conversion. Staking will be detailed in another design doc and may change this ratio.  Upon receiving Capacity, it can then be used to pay for transactions. 
```